cluster-scope/base/rbac.authorization.k8s.io/clusterroles/xdmod-permissions/clusterrole.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: xdmod-permissions
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
```

Comment:
Does xdmod make use of the watch feature as described in https://kubernetes.io/docs/reference/using-api/api-concepts/#efficient-detection-of-changes (which would require adding the watch verb as well)?

Reply:
Not that I know of - at least not the part I'm working on that generates the log files. @rob-baron may be able to answer if the main app uses the watch feature.

Reply:
xdmod doesn't know anything about kubernetes or openstack.  All knowledge of either platform is contained in the scripts that we (mainn and I are writing).  The watch feature would be an excellent addition to the xdmod-openshift script if it simplifies fetching only the data that has changed.